src/main/java/pl/poznan/put/jsontools/logic/JsonTransformMinify.java

```diff
@@ -1,15 +1,23 @@
 package pl.poznan.put.jsontools.logic;
 
 
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
 
 public class JsonTransformMinify extends JsonTransformer {
     public JsonTransformMinify(JsonTransform transform) {
         super(transform);
     }
-
+    private static final ObjectMapper mapper = new ObjectMapper();
     @Override
     public String execute() {
+        try {
+            return mapper.readTree(super.execute()).toString();
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
Throw InvalidJson exception instead, so Spring will handle it.
```suggestion
            Throw new JsonToolsInvalidJsonError(e.getMessage);
```